src/governance/IArtemisGovernor.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity 0.8.19;
+
+import {IWormhole} from "@protocol/wormhole/IWormhole.sol";
+
+/// @notice interface for the Artemis Governor Contract
+interface IArtemisGovernor {
```

Comment:
This interface is not being used anywhere. Interface in interfaces folder is used. we can delete this.